docs-ref-conceptual/get-started-tutorial-1-prepare-environment.md

```diff
@@ -0,0 +1,223 @@
+
+* Use the Bash environment in [Azure Cloud Shell](/azure/cloud-shell/overview) by selecting the **Open Cloud Shell** button in the top right corner of each Azure CLI code block.
+
+* If you prefer to run the Azure CLI reference commands locally, [install](/cli/azure/install-azure-cli) the Azure CLI.
+
+* To complete the steps in this tutorial, you need `contributor` permissions in your subscription.
+
+[!INCLUDE [include](./includes/current-version.md)]
+
+## Sign in to Azure using the Azure CLI
+
+There are several [authentication options](./authenticate-azure-cli.md) when working with the Azure CLI. The Azure CLI's default authentication method for logins uses a web browser and access token to sign in.
+
+[!INCLUDE [interactive_login](includes/interactive-login.md)]
+
+## Find and change your active subscription
+
+After logging into the Azure CLI, always check your current subscription. If you aren't working under the subscription you prefer, change your subscription using [az account set](/cli/azure/account#az-account-set). Here's a code example to use.
+
+```azurecli-interactive
+# see your current/default subscription
+az account show
+
+# find the list of subscriptions available to you
+az account list --output table
+
+# change your current/default subscription
+az account set --subscription <mySubscriptionName>
+
+# you can also set your subscription using a subscription ID
+az account set --subscription <00000000-0000-0000-0000-000000000000>
+```
+
+## Learn about resource groups
+
+A resource group is a container for Azure resources. To create a resource group, you need role permissions of `contributor` or higher in your subscription.
+
+### Create a resource group
+
+1. A resource group name must be globally unique. First check if the name you want is available using the [az group exists](/cli/azure/group#az-group-exists) command. An output value of `false` means that the name is available.
+
+   ```azurecli-interactive
+   az group exists --name <myUniqueRGname>
+   ```
+
+1. Retrieve a list of supported regions for your subscription with the [az account list-locations](/cli/azure/account#az-account-list-locations) command. For a list of available Azure locations, see [Choose the right Azure region for you](https://azure.microsoft.com/explore/global-infrastructure/geographies/#overview).
+
+   ```azurecli-interactive
+   az account list-locations --query "[].{Region:name}" --output table
+   ```
+
+1. It's time to create your resource group! Use the [az group create](/cli/azure/group#az-group-create) command.
+
+   ```azurecli-interactive
+   az group create --location <myLocation> --name <myUniqueRGname>
+   ```
+
+### Use a random ID to create a resource group
+
+When testing, it's best to create a resource group that contains a random ID. Using a random ID allows you to retest your code without having to wait for a prior resource group of the same name to be removed from Azure.
+
```

Comment:
Should there be heading for the section "Bash and PowerShell variable syntax", or is it part of the Random ID section?